src/Component/Sender/Sender.php

```diff
@@ -44,8 +45,15 @@ public function __construct(
     /**
      * {@inheritdoc}
      */
-    public function send(string $code, array $recipients, array $data = [], array $attachments = [], array $replyTo = []): void
-    {
+    public function send(
```

Comment:
In this case, we should probably add some deprecation notice and docblock about the possibility of two additional fields. Probably even on interface

Reply:
Done here: https://github.com/Sylius/SyliusMailerBundle/pull/152/files#diff-e3fec7c4efcfcc0a820a2f6ba9a6926512d60b73468f5a38cf6fe674bbcf0483R19